### PR TITLE
[T2A2][F12-A3]Gabriel Sim

### PR DESCRIPTION
**Pull request by gabrielsim**
Ready for review

### CODE REVIEW

**Comment by gabrielsim1993** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -794,11 +794,11 @@ private static void addPersonToAddressBook(String[] person) {
      * @return true if the given person was found and deleted in the model
      */
     private static boolean deletePersonFromAddressBook(String[] exactPerson) {
-        final boolean changed = ALL_PERSONS.remove(exactPerson);
-        if (changed) {
+        final boolean isChanged = ALL_PERSONS.remove(exactPerson);
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:797]: good catch!

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1144,14 +1144,10 @@ private static String getUsageInfoForExitCommand() {
...
     /**
-     * Removes sign(p/, d/, etc) from parameter string
-     *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
-     */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    * Removes prefix from the given fullString if prefix occurs at the start of the string.
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1147]: :+1:

**Review by okkhoy — COMMENTED** [review]
please pay attention to additional code improvements you can make like variable and method names.

**Comment by okkhoy** [comment]
@gabrielsim1993 some comments added. please take a look, ack and close the PR.

**Comment by gabrielsim** [comment]
Noted with thanks!

**Pull request closed without merging**